Google.PowerShell/ReleaseFiles/GoogleCloudPlatform.Format.ps1xml

```diff
@@ -162,5 +162,52 @@
         </TableRowEntries>
       </TableControl>
     </View>
+    <View>
+      <Name>BigQuery Data Row</Name>
```

Comment:
That's weird. I'm fine if we fix it in another check-in though. Can you add a TODO for it?